test/g_searcher_web/validators/create_report_params_test.exs

```diff
@@ -0,0 +1,44 @@
+defmodule GSearcher.Validators.CreateReportParamsTest do
+  use GSearcher.DataCase
+
+  alias GSearcherWeb.Validators.CreateReportParams
+
+  describe "changeset/2" do
+    test "returns valid changeset given valid params" do
+      report = build(:report)
+
+      params = %{
+        "title" => "Test Report",
+        "csv" => %Plug.Upload{
+          path: report.csv_path,
+          filename: "test.csv",
+          content_type: "text/csv"
+        }
+      }
+
+      changeset = CreateReportParams.changeset(params)
+
+      assert changeset.valid? === true
+    end
+
+    test "returns invalid chanegset given invalid params" do
+      params = %{
+        "title" => "",
+        "csv" => %Plug.Upload{
+          path: "/cat_picture.jpg",
+          filename: "cat_picture.jpg",
+          content_type: "image/jpeg"
+        }
+      }
+
+      changeset = CreateReportParams.changeset(params)
+
+      assert changeset.valid? === false
```

Comment:
Just wondering  if using `===` affected to int and float only 🤔  

```
The difference between == and === is that the latter is more strict when comparing integers and floats:
```

https://elixir-lang.org/getting-started/basic-operators.html